### PR TITLE
2.0 API: device_guard

### PR DESCRIPTION
**Pull request by zhangting2020**
2.0 API: device_guard
image

### CODE REVIEW

File: doc/paddle/api/paddle/fluid/framework/device_guard_cn.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. _cn_api_device_guard:
+
+device_guard
+-------------------------------
+
+**注意：该API仅支持静态图模式**
+
+.. py:function:: paddle.fluid.device_guard(device=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/framework/device_guard_cn.rst]: 这里直接写成 paddle.device_guard 好一点~

Comment (zhangting2020) [reply doc/paddle/api/paddle/fluid/framework/device_guard_cn.rst]: done.

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**